test/test_heuristics.rb

```diff
@@ -100,7 +112,7 @@ def test_bb_by_heuristics
 
   def test_builds_by_heuristics
     assert_heuristics({
-      "Text" => all_fixtures("Text"),
+      nil => all_fixtures("Text"),
```

Comment:
Not sure I understand this change. Why did it change?

Reply:
[See below](https://github.com/github/linguist/pull/4936#discussion_r479645840). Classifying `builds` files as `Text` was presumably somebody's attempt at marking them as having no language. Now we have a more appropriate means to do that.